sbws/lib/relaylist.py

```diff
@@ -107,6 +131,48 @@ def master_key_ed25519(self):
             return None
         return key.rstrip('=')
 
+    @property
+    def consensus_valid_after(self):
+        network_status_document = self._from_ns('document')
+        if network_status_document:
+            return getattr(network_status_document, 'valid_after', None)
+        return None
+
+    @property
+    def last_consensus(self):
```

Comment:
documentation

Reply:
Should this be called last_consensus_timestamp or last_consensus_valid_after?  With the current name, it sounds like it returns a consensus rather than a timestamp.